src/routes/sessions/[uuid]/.Requester/.Requester.svelte

```diff
@@ -55,8 +54,19 @@
 </div>
 
 <div class="is-flex is-justify-content-space-evenly">
-  {#each sipQueues as { extensionNumber, title }, index}
-    <QueuePrompt {index} {uuid} {isSDK} {isIC} {isSIP} {extensionNumber} {title} {embeddable} on:notif={showNotif} />
+  {#each sipQueues as { extensionNumber, title, sipImage }, index}
+    <QueuePrompt
+      {index}
+      {uuid}
+      {isSDK}
+      {isIC}
+      {isSIP}
```

Comment:
Would be helpful for others to rename these variables and all related columns/fields into something that's more intuitive. Like:
- `isSdkBased` `isInstantConnectBased`, `isSipBased`
- `isSdkEnabled` `isInstantConnectEnabled`, `isSipEnabled`
-  etc.

The reason being, that the selection is multiple-choice and not unitary.